mq/main/pom.xml

```diff
@@ -140,22 +140,22 @@
             <dependency>
                 <groupId>jakarta.resource</groupId>
                 <artifactId>jakarta.resource-api</artifactId>
-                <version>2.0.0-RC1</version>
+                <version>2.0.0</version>
             </dependency>
             <dependency>
                 <groupId>jakarta.servlet</groupId>
                 <artifactId>jakarta.servlet-api</artifactId>
-                <version>5.0.0-M1</version>
+                <version>5.0.0</version>
```

Comment:
In GF: [nucleus/parent/pom.xml](https://github.com/eclipse-ee4j/glassfish/blob/d6f0dfa7adf823908d95bfc3e9490ec244d509ee/nucleus/parent/pom.xml#L89)